jenkins/shared-libraries/linux/vars/ExecuteCodeQL.groovy

```diff
@@ -147,11 +151,25 @@ def call(org, repo, branch, language, buildCommand, token, installCodeQL) {
         fi
         echo "Database initialized"
 
+        echo "WORKSPACE: ${WORKSPACE}"
+        echo "PWD: \${PWD}"
+        echo "Check if PWD matches WORKSPACE"
```

Comment:
Can we make these natural language as well in the follow on PR, the last line should be `Check if current working directory and Jenkins workspace are the same directory`